src/main/java/seedu/address/model/person/Person.java

```diff
@@ -111,6 +147,7 @@ public boolean isSamePerson(Person otherPerson) {
 
         return otherPerson != null
                 && otherPerson.getName().equals(getName())
+                && otherPerson.getNric().equals(getNric())
```

Comment:
Should we only keep NRIC equality and leave out the rest? Since NRIC already uniquely identifies the person

Reply:
yea we can, but don't merge yet i'm not done haha i committed to make sure everything is working normally first. I haven't done the following:

1. Make sure NRIC is unique against a database of NRICs
2. Make JobsApply compulsory 
3. Make InterviewScores optional

Reply:
Oh alright, thanks for informing. I was trying to look through for the unique NRIC checking.

On a side note, technically NRIC also has a check sum modulo (11 or 13 I cant rmb) that sums all the digits then modulo them to get a number with a letter assigned to each number in that modulo which gives the last letter in the NRIC

Reply:
But that one can don't need do, considering we got some many things already hahaha.

Reply:
LOL WHAT that's how the last letter works??????

Reply:
i thought it was random lmao

Reply:
Yea, you can go google NRIC check sum to learn more, can use your own NRIC to see if it's really true

Reply:
yea it works